src/charts/CytoViz/components/DetailsLog/DetailsLog.js

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import useStyles from './style';
+
+function prettyPercentage(val) {
+  return (val * 100).toFixed(1) + '%';
+}
+
+function hiddenElements(cytoElements) {
+  let hiddenElements = 0;
+  cytoElements.forEach((element) => {
+    if (element.data('hidden')) {
+      hiddenElements++;
+    }
+  });
+  return hiddenElements;
+}
+
+const DetailsLog = (props) => {
+  const { cytoAsState } = props;
+  const classes = useStyles();
+
+  const compounds = cytoAsState.nodes().parent().length;
+  const children = cytoAsState.nodes().nonorphans().length;
+  const noCompounds = cytoAsState
```

Comment:
In a graph, nodes that have no children are called `leaves` (as the leaves of trees), it may be a better name than `noCompounds` :smile:

Reply:
Yes, `leaves` is a good name for the nodes described stored in this variable. Nevertheless, it could be misleading. 

Generally this property refers to the topological structure, not the hierarchical structure of the network. Taking the brewery as an example, we have the topological structure: 
- Brewery --> Customer1 -->Customer4, when thinking of a tree. Here, only the Customer4 would be a `leave`. 
In the hierarchical structure instead is: 
- Brewery -countains-> Customers.
 
So all Customers would be leaves, which is my desired result. What do you think about something like: `notCompoundNodes`